2021-12-08 03:24:00,391 INFO    MainThread:72384 [wandb_setup.py:_flush():71] setting env: {}
2021-12-08 03:24:00,391 INFO    MainThread:72384 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-08 03:24:00,391 INFO    MainThread:72384 [wandb_init.py:_log_setup():371] Logging user logs to /Users/sidharthmenon/workspace/classes/6.887/gyoza/wandb/run-20211208_032400-2jnby3cs/logs/debug.log
2021-12-08 03:24:00,391 INFO    MainThread:72384 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/sidharthmenon/workspace/classes/6.887/gyoza/wandb/run-20211208_032400-2jnby3cs/logs/debug-internal.log
2021-12-08 03:24:00,391 INFO    MainThread:72384 [wandb_init.py:init():404] calling init triggers
2021-12-08 03:24:00,391 INFO    MainThread:72384 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-12-08 03:24:00,392 INFO    MainThread:72384 [wandb_init.py:init():449] starting backend
2021-12-08 03:24:00,392 INFO    MainThread:72384 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-08 03:24:00,405 INFO    MainThread:72384 [backend.py:ensure_launched():199] starting backend process...
2021-12-08 03:24:00,414 INFO    MainThread:72384 [backend.py:ensure_launched():205] started backend process with pid: 72414
2021-12-08 03:24:00,420 INFO    MainThread:72384 [wandb_init.py:init():458] backend started and connected
2021-12-08 03:24:00,425 INFO    MainThread:72384 [wandb_init.py:init():519] updated telemetry
2021-12-08 03:24:00,461 INFO    MainThread:72384 [wandb_init.py:init():549] communicating current version
2021-12-08 03:24:01,449 INFO    MainThread:72384 [wandb_init.py:init():554] got version response 
2021-12-08 03:24:01,449 INFO    MainThread:72384 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-08 03:24:01,594 INFO    MainThread:72384 [wandb_init.py:init():592] starting run threads in backend
2021-12-08 03:24:01,766 INFO    MainThread:72384 [wandb_run.py:_console_start():1816] atexit reg
2021-12-08 03:24:01,767 INFO    MainThread:72384 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-08 03:24:01,768 INFO    MainThread:72384 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-08 03:24:01,770 INFO    MainThread:72384 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-08 03:24:01,771 INFO    MainThread:72384 [wandb_init.py:init():619] run started, returning control to user process
2021-12-08 03:24:01,772 INFO    MainThread:72384 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-08 03:24:01,772 INFO    MainThread:72384 [wandb_run.py:_restore():1758] restore
2021-12-08 03:24:04,281 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 1046
}

2021-12-08 03:24:04,468 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 1046
}

2021-12-08 03:24:04,611 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 4088
}

2021-12-08 03:24:04,718 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1046
  total_bytes: 4088
}

2021-12-08 03:24:04,822 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2021-12-08 03:24:04,927 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2021-12-08 03:24:05,030 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2021-12-08 03:24:05,132 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2021-12-08 03:24:05,238 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2021-12-08 03:24:05,665 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}

2021-12-08 03:24:05,945 INFO    MainThread:72384 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4088
  total_bytes: 4088
}
local_info {
}

2021-12-08 03:24:07,078 INFO    MainThread:72384 [wandb_run.py:_append_files():2184] logging synced files
